setup.cfg

```diff
@@ -0,0 +1,68 @@
+[metadata]
+name = jaxfit
+description = A proof-of-concept for custom minimizers
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/lgray/jaxfit
+author = Nick Smith
+author_email = nick.smith@cern.ch
+maintainer = Nick Smith
+maintainer_email = nick.smith@cern.ch
+license = BSD-3-Clause
+license_file = LICENSE
+platforms =
+    Any
+classifiers =
+    Development Status :: 1 - Planning
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+project_urls =
+    Documentation = https://jaxfit.readthedocs.io/
+    Bug Tracker = https://github.com/lgray/jaxfit/issues
+    Discussions = https://github.com/lgray/jaxfit/discussions
+    Changelog = https://github.com/lgray/jaxfit/releases
+
+[options]
+packages = find:
+install_requires =
```

Comment:
Ideally install requires just jax and jaxlib. Tests may require all the above, and more (root...)

Reply:
Fixed.